packages/token-lists/react/lists.ts

```diff
@@ -21,60 +21,54 @@ export const createListsAtom = (storeName: string, reducer: any, initialState: a
    * Persist you redux state using IndexedDB
    * @param {string} dbName - IndexedDB database name
    */
-  function IndexedDBStorage(dbName: string) {
+  function IndexedDBStorage<Value>(dbName: string): AsyncStorage<Value> {
     if (typeof window !== 'undefined') {
       const db = localForage.createInstance({
         name: dbName,
         storeName,
       })
       return {
-        db,
         getItem: async (key: string) => {
           const value = await db.getItem(key)
           if (value) {
             return value
           }
           return initialState
         },
-        setItem: (k: string, v: any) => {
+        setItem: async (k: string, v: any) => {
```

Comment:
no need to change here maybe? As the `db.setItem` is returning a promise

Reply:
actually, it is a promise

https://github.com/localForage/localForage/blob/master/typings/localforage.d.ts#L20

Reply:
follow the `AsyncStorage` type defined, the return type of `setItem` is `Promise<void>`, so no need to return

https://github.com/pmndrs/jotai/blob/1d25284f170275186e0ff1715967c7e560debae6/src/vanilla/utils/atomWithStorage.ts#L17

Reply:
follow the `AsyncStorage` type defined, the return type of `setItem` is `Promise<void>`, so no need to return

https://github.com/pmndrs/jotai/blob/1d25284f170275186e0ff1715967c7e560debae6/src/vanilla/utils/atomWithStorage.ts#L17